content/en/docs/ops/ambient/migrate/index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Migrating from Sidecars to Ambient
+description: How to migrate existing sidecar topology to Ambient
+weight: 1
```

Comment:
Do we want this Migration doc to really be the first doc in the list?

Reply:
As noted, we need to update the weight to reflect the correct position on the page. Should it be after `User Guides` (so a number greater than 15 and less than 20, or should this be under `User Guides`. I think I favor not being a User Guide.